### PR TITLE
[mod_menu]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_menu
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_menu/mod_menu.xml [hunk 1]
```diff
@@ -77,70 +80,77 @@
...
 					label="COM_MODULES_FIELD_CACHE_TIME_LABEL"
-					description="COM_MODULES_FIELD_CACHE_TIME_DESC" />
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
+					default="900"
+				/>
+
 				<field
 					name="cachemode"
 					type="hidden"
 					default="itemid">
```
Comment (andrepereiradasilva) [inline modules/mod_menu/mod_menu.xml]: shouldn't the > be in a new line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 8a13cd4a8fa6be68bc21359bf6c50a6f391bd1da<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10587">issues.joomla.org/joomla-cms/10587</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 8a13cd4a8fa6be68bc21359bf6c50a6f391bd1da<br /><br />on review - Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10587">issues.joomla.org/joomla-cms/10587</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10587">issues.joomla.org/joomla-cms/10587</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 8a13cd4a8fa6be68bc21359bf6c50a6f391bd1da<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10587">issues.joomla.org/joomla-cms/10587</a>.</sub>

**Pull request closed without merging**